benchmark/storage/data_table_benchmark.cpp

```diff
@@ -88,7 +88,7 @@ BENCHMARK_DEFINE_F(DataTableBenchmark, SimpleInsert)(benchmark::State &state) {
     storage::DataTable table(&block_store_, layout_, storage::layout_version_t(0));
     // We can use dummy timestamps here since we're not invoking concurrency control
     transaction::TransactionContext txn(transaction::timestamp_t(0), transaction::timestamp_t(0), &buffer_pool_,
-                                        LOGGING_DISABLED);
+                                        LOGGING_DISABLED, nullptr);
```

Comment:
nit: Probably best to be consistent and use a MACRO that is more meaningful

Reply:
Fixed